jax/_src/interpreters/mlir.py

```diff
@@ -2228,9 +2228,14 @@ def map_compute_type(c_type):
 
 def wrap_compute_type_in_place(ctx, op):
```

Comment:
Do you also need to change `_platforms_for_eqn_ctx` function and use `gpu` as the platform for that equation?

Reply:
I think we should be ok since right now that method defaults to returning `()`, https://github.com/jax-ml/jax/blob/47858c4ac2fd4757a3b6fc5bb2981b71a71f00c2/jax/_src/interpreters/mlir.py#L1942 which works ok based on my testing.